pkgs/applications/editors/brackets/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchurl, buildEnv, gtk, glib, gdk_pixbuf, alsaLib, nss, nspr, gconf
+, cups, libgcrypt_1_5, makeWrapper, dbus, udev }:
+let
+  bracketsEnv = buildEnv {
+    name = "env-brackets";
+    paths = [
+      gtk glib gdk_pixbuf stdenv.cc.cc alsaLib nss nspr gconf cups libgcrypt_1_5
+      dbus udev
+      /* zlib    atk pango freetype libgnome_keyring3
+      fontconfig  cairo  expat libgpgerror
+      xorg.libXrender xorg.libX11 xorg.libXext xorg.libXdamage xorg.libXtst
+      xorg.libXcomposite xorg.libXi xorg.libXfixes xorg.libXrandr
+      xorg.libXcursor libcap */
+    ];
+  };
+in
+stdenv.mkDerivation rec {
+  name = "brackets-${version}";
+  version = "1.5";
+
+  src = fetchurl {
+    url = "https://github.com/adobe/brackets/releases/download/release-${version}/Brackets.Release.${version}.64-bit.deb";
+    sha256 = "1fc8wvh9wbcydd1sw20yfnwlfv7nllb6vrssr6hgn80m7i0zl3db";
+    name = "${name}.deb";
+  };
+
+  phases = [ "installPhase" ];
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out
+    ar p $src data.tar.xz | tar -C $out -xJ
+
+    mv $out/usr/* $out/
+    rmdir $out/usr
+    ln -sf $out/opt/brackets/brackets $out/bin/brackets
+
+    ln -s ${udev}/lib/libudev.so.1 $out/opt/brackets/lib/libudev.so.0
```

Comment:
nope, it does not work without it

without that line it returns this error:
```
/nix/store/4ajf4kpv7rwwq0wfafapryap43nd7n56-brackets-1.5/opt/brackets/.brackets-wrapped: error while loading shared libraries: libudev.so.0: cannot open shared object file: No such file or directory
```

and those issues indicate that linking from 1 to 0 is the correct way

https://github.com/adobe/brackets/issues/4720
https://github.com/adobe/brackets/issues/4816
...